airbyte-integrations/connectors/source-looker/source_looker/manifest.yaml

```diff
@@ -0,0 +1,1120 @@
+version: 0.83.0
```

Comment:
Make sure you actually run this in your local OSS builder. Specifically, I expect this `version` key to be the latest available CDK, i.e. 3.9.0, and the structure of the yaml might be slightly different. 

I'm happy to make a loom explaining how to run the platform + Builder with custom components support locally to test this out.

Reply:
> I'm happy to make a loom

That would be great, especially for using custom components with the builder locally

Reply:
I'm on it — turns out our local setup is slightly broken, working to fix it and make everything work with `abctl`.